brayns/common/utils/ImageUtils.cpp

```diff
@@ -22,6 +22,8 @@
 
 #include <brayns/common/renderer/FrameBuffer.h>
 
+#ifdef BRAYNS_USE_FREEIMAGE
```

Comment:
This is not user guide 😄

Reply:
That's why this commit is called "Fix build without FreeImage" ;-)

Reply:
I was looking at the PR name 😄

Reply:
I'm not going to squash the commits, and I don't want to open a separate PR for such a small fix...